schema/spacewalk/postgres/start.sql

```diff
@@ -13,5 +13,3 @@
 -- in this software or its documentation.
 --
 
-create extension dblink;
-CREATE EXTENSION pgcrypto;
```

Comment:
I am not sure if we can drop this, since we have some usage of hashing function that this pgcrypto provides: https://www.postgresql.org/docs/current/pgcrypto.html#PGCRYPTO-GENERAL-HASHING-FUNCS-DIGEST

Reply:
Are you sure we are using postgresql digest and not something from java? I tried to grep the source and I do not see it.
Particularly since this was added in https://github.com/uyuni-project/uyuni/pull/7531 and since then removed in https://github.com/uyuni-project/uyuni/pull/8045 except that `create extension` part.

Reply:
None of the functions in that doc is actually used in the `schema` folder. I used this fish shell script to check:
```
for f in "digest" "hmac" "crypt" "gen_salt" "pgp_sym_encrypt" "pgp_sym_decrypt" "pgp_pub_encrypt" "pgp_pub_decrypt" "pgp_key_id" "armor" "pgp_armor_headers" "encrypt" "decrypt" "encrypt_iv" "decrypt_iv" "gen_random_bytes" "gen_random_uuid"
      rg -i $f'[[:space:]]*\('
  end
  ```

Reply:
@aaannz you are correct, I just run a grep and saw one usage but it's only used in a migration script [1], and that index is dropped in a latter update script. 
Thank you for the commit pointers.

[1] https://github.com/uyuni-project/uyuni/blob/master/schema/spacewalk/upgrade/susemanager-schema-4.3.21-to-susemanager-schema-4.3.22/203-add-changelogdata-index.sql#L36